File: src/primus/pawel/Exercise_020.java
package primus.pawel;

// Counting Duplicates
// 6kyu
// https://www.codewars.com/kata/54bf1c2cd5b56cc47f0007a1/train/java
// 2021-03-16 T:18:21:44
class Exercise_020 {

    public static int duplicateCount(String text) {
        text = text.toUpperCase();
        int result = 0;
        int[] resultTab = new int[126];
        for (int i = 0; i < text.length(); i++) {
            //System.out.println((int)text.charAt(i));
            resultTab[(int) text.charAt(i)]++;
        }

        for (int value : resultTab) {
            if (value > 1) {
                result++;
            }
        }
        return result;
    }

    public static void main(String[] args) {

        System.out.println(duplicateCount("indivisibility"));

    }


}

File: src/JS/Exercise_020.js

// Beginner - Lost Without a Map
// 8kyu
// https://www.codewars.com/kata/57f781872e3d8ca2a000007e/train/java
// 2022-03-23 T:21:45:38

function maps(x) {
    let arr = Array.from(x)
    let doubleArray = new Array();

    for (let i = 0; i < arr.length; i++) {
        doubleArray[i] = arr[i]*2;

    }
    return doubleArray;
}

//function maps(x){
//   return x.map(n => n * 2);
// }